docs/AI-for-security/connect-to-vertex.asciidoc

```diff
@@ -0,0 +1,119 @@
+[[connect-to-vertex]]
+= Connect to Google Vertex
+
+:frontmatter-description: Set up a Google Vertex LLM connector.
+:frontmatter-tags-products: [security]
+:frontmatter-tags-content-type: [guide]
+:frontmatter-tags-user-goals: [get-started]
+
+This page provides step-by-step instructions for setting up a Google Vertex AI connector for the first time. This connector type enables you to leverage Vertex AI's large language models (LLMs) within {elastic-sec}. You'll first need to enable Vertex AI, then generate an API key, and finally configure the connector in your {elastic-sec} project.
+
+IMPORTANT: Before continuing, you should have an active project in one of Google Vertex AI's https://cloud.google.com/vertex-ai/docs/general/locations#feature-availability[supported regions]. 
+
+[discrete]
+== Enable the Vertex AI API
+
+1. Log in to the GCP console and navigate to **Vertex AI → Vertex AI Studio → Overview**. 
+2. If you're new to Vertex AI, the **Get started with Vertex AI Studio** popup appears. Click **Vertex AI API**, then click **ENABLE**.
+
+The following video demonstrates these steps.
+
+=======
+++++
+<script type="text/javascript" async src="https://play.vidyard.com/embed/v4.js"></script>
+<img
+  style="width: 100%; margin: auto; display: block;"
+  class="vidyard-player-embed"
+  src="https://play.vidyard.com/vFhtbiCZiKhvdZGy2FjyeT.jpg"
+  data-uuid="vFhtbiCZiKhvdZGy2FjyeT"
+  data-v="4"
+  data-type="inline"
+/>
+</br>
+++++
+=======
+
+NOTE: For more information about enabling the Vertex AI API, refer to https://cloud.google.com/vertex-ai/docs/start/cloud-environment[Google's documentation].
+
+[discrete] 
+== Create a Vertex AI service account
+
+1. In the GCP console, navigate to **APIs & Services → Library**.
+2. Search for **Vertex AI API**, select it, and click **MANAGE**.
+3. In the left menu, navigate to **Credentials** then click **+ CREATE CREDENTIALS** and select **Service account**.
```

Comment:
Similar preference: when UI labels include extraneous typographical symbols like `+` or `>>`, I tend to omit them. They seem distracting in docs and usually aren't necessary for readers to understand which button you're referring to.